Resources/Prototypes/Entities/Structures/Doors/Windoors/base_structurewindoors.yml

```
@@ -183,11 +194,16 @@
     - state: panel_open
       map: [ "enum.WiresVisualLayers.MaintenancePanel" ]
       visible: false
-    - state: electrified
+    - state: electrified_ai
       sprite: Interface/Misc/ai_hud.rsi
       shader: unshaded
       visible: false
-      map: [ "enum.ElectrifiedLayers.HUD" ]
+      map: ["enum.ElectrifiedLayers.HUD"]
```

```suggestion
      map: [" enum.ElectrifiedLayers.HUD "]
```
make it consistent throughout the prototype one way or the other